Raylib-cs/types/Window.cs

```diff
@@ -0,0 +1,167 @@
+﻿using System;
+using System.Numerics;
+
+namespace Raylib_cs;
+
+public sealed class Window : IDisposable
+{
+    private static Window _instance;
+    
+    private Window(ConfigFlags flags, int width, int height, string title)
+    {
+        Raylib.SetConfigFlags(flags);
+        Raylib.InitWindow(width, height, title);
+        _instance = this;
+    }
+    
+    /// <summary>
+    /// Gets an instance of the Window or creates a new one if it doesn't exist.
+    /// </summary>
+    /// <param name="flags">The configuration flags for the window.</param>
+    /// <param name="width">The width of the window.</param>
+    /// <param name="height">The height of the window.</param>
+    /// <param name="title">The title of the window.</param>
+    /// <returns>An instance of the Window.</returns>
+    public static Window Instance(ConfigFlags flags, int width, int height, string title)
+    {
+        return _instance ?? new Window(flags, width, height, title);
+    }
```

Comment:
I think it would be better to rename this to Init and make it so we can open the window again if we closed it before.